Aire/View/NibClass/SceneView.swift

```diff
@@ -20,10 +20,25 @@ class SceneView: NibView {
 	private let aqiToCountMultiplier = 1.5
 	// At least these n individual pollutants should be close to viewer
 	private let nClosePollutants: Int = 20
+    
+    func renderer(_ renderer: SCNSceneRenderer, updateAtTime time: TimeInterval) {
+        if let camera = sceneView.session.currentFrame?.camera {
+            // position of camera
+            let transform = camera.transform
```

Comment:
mmm not sure if we need the data type: simd_float4x4, since it's a constant value, we don´t need to cast it to any data type itself, I can name it more descriptive